src/CodeGeneration.Roslyn.Tasks/CodeGeneration.Roslyn.Tasks.csproj

```diff
@@ -27,4 +29,27 @@
     <PackageReference Include="Nerdbank.MSBuildExtension" Version="0.1.17-beta" PrivateAssets="all" />
   </ItemGroup>
 
+  <Target Name="SetPackageVersionInProps" DependsOnTargets="GetBuildVersion">
+    <PropertyGroup>
+      <PropsFileDestinationDirectory>$(IntermediateOutputPath)build\</PropsFileDestinationDirectory>
+      <PropsFilePath>$(PropsFileDestinationDirectory)\$(PropsFileName)</PropsFilePath>
+      <_ProjectNamespaces>
+        <Namespace Prefix="dn" Uri="http://schemas.microsoft.com/developer/msbuild/2003" />
+      </_ProjectNamespaces>
+    </PropertyGroup>
+    <MakeDir Directories="$(PropsFileDestinationDirectory)" />
+    <Copy SourceFiles="build\$(PropsFileName)" DestinationFiles="$(PropsFilePath)" />
+    <XmlPoke XmlInputPath="$(PropsFilePath)" Value="$(PackageVersion)" Query="/dn:Project/dn:ItemGroup/dn:DotNetCliToolReference/@Version" Namespaces="$(_ProjectNamespaces)" />
+    <ItemGroup>
+      <None Remove="build\$(PropsFileName)" />
+      <None Include="build\$(PropsFileName)">
+        <Pack>false</Pack>
+      </None>
+      <None Include="$(PropsFilePath)">
+        <Pack>true</Pack>
+        <PackagePath>build\</PackagePath>
+      </None>
+    </ItemGroup>
+    <Message Text="None now includes: @(None->'%(Identity) Pack:%(Pack) PackagePath:%(PackagePath)')" Importance="high"/>
```

Comment:
I'm guessing this `<Message>` was for your own diagnostic purposes and shouldn't be retained in the PR.

Reply:
Indeed. Will be removed.